### DISCUSSION TITLE
How to plug this into a CI/CD pipeline?

### DISCUSSION THREAD

**Discussion by SwathiMystery** [aws-samples/aws-saas-factory-eks-reference-architecture#46] [Ideas]
Thank you for providing this reference architecture. It is extremely helpful.

One question though, what would be the right mechanism to plug this into CI/CD pipeline?
Although I understand that setup/deploy scripts will be run before that.

Any pointers will be really helpful. 

Thanks 🙏

**Comment by tobuck-aws** [comment]
HI @SwathiMystery ,
I'm not sure what you're referring to? Do you mean plugging all of the various services into a CI/CD pipeline, so they get automatically deployed as part of an update? Or something else? Converting this to a discussion. We can definitely create an issue if we need to.

**Reply by SwathiMystery** [reply]
@tobuck-aws Sorry for the cryptic message. 

Yes, you are right - I mean plugging all of the various services into a CI/CD pipeline, so they get automatically deployed/updated as part of every update to the code?
Thanks for your response 🙏

**Comment by ranjith-aws** [comment]
Hi @SwathiMystery Thanks for that additional context. So we actually have something in the works to support that where updates made to the code would re-deploy the services. We have not established a timeline for it yet, but definitely something in our radar and already being worked upon. Appreciate you bringing it up.

**Reply by SwathiMystery** [reply]
Thanks for the quick response @ranjith-aws 🙏 😊 
Any high level context about how we can achieve the same and also with various environments like dev, staging, prod?

Thanks for this jump start repository - really helpful 🙏 🙏 🙏
